src/canvas/asciicanvas.jl

```diff
@@ -111,6 +111,8 @@ struct AsciiCanvas <: LookupCanvas
     origin_y::Float64
     width::Float64
     height::Float64
+    xscale::Symbol
+    yscale::Symbol
```

Comment:
I know this is Plots convention to use symbols so no objections, but just let me raise a note here for this design: from a computational perspective, any scalar function can be put here (say, `abs2` or more complicated coordinate transformations).

Reply:
This is now fixed, and tested.